configure

```diff
@@ -1742,6 +1742,7 @@ EXAMPLE_LIST="
     scaling_video_example
     transcode_aac_example
     transcoding_example
+    vaapi_decode_example
```

Comment:
Can we use current hw decode example to do this test?

Reply:
If use hw_deocde do this, it will need add additional code for sub frame in hw_decode that will only available for vaapi. To avoid the community say that change will only for vaapi. So I separate it from hw_decode. I think we can send current method to community. If community doesn't mind to add sub frame code in hw_decode, I can change it back to hw_decode.